src/data/equiv/algebra.lean

```diff
@@ -356,3 +356,124 @@ instance symm.is_ring_hom {e : α ≃r β} : is_ring_hom e.to_equiv.symm := hom
   e.symm.to_equiv x = e.to_equiv.symm x := rfl
 
 end ring_equiv
+
+
+/-
+We define the type of automorphisms on groups, additive monoids,
+additive groups and rings using `mul_equiv`, `add_equiv` and `ring_equiv`.
+In each case this type also forms a group.
+-/
+
+@[extensionality] lemma mul_equiv.ext {α β : Type*} [has_mul α] [has_mul β]
```

Comment:
Move closer to the definition of `mul_equiv`. This will get lost here.

Reply:
fixed